Week-10/url-shortener/src/components/Header/style.css

```diff
@@ -0,0 +1,33 @@
+.headerContainer {
+  display: flex;
+  height: 80px;
+  width: 100%;
+  justify-content: space-between;
```

Comment:
create a mixin at top level and reuse it across instead of writing same style in multiple css files